### ISSUE TITLE
No Connection between Samsung Note8 and LM19.3 Cinnamon

### ISSUE THREAD

**Issue by jtcraigjr** [linuxmint/warpinator#82]
I have a thread running that details all I have done.

https://forums.linuxmint.com/viewtopic.php?f=47&t=349343

the 2nd page is probably most informative, but I have posted Screenshots of the cellphone and my LM19.3 Cinnamon laptop.

Same Group Code on phone and laptop:  Warpinator on port 42000

**Comment by mtwebster** [comment]
This is implying you have a connection, but maybe I'm taking it out of context:
![image](https://user-images.githubusercontent.com/262776/118546498-09716a00-b726-11eb-8c4a-fc26ed5c77ca.png)

You should not be able to send files in **either** direction until there is a full 2-way connection, so I'm not sure what's going on there. I know absolutely nothing about the android app - we don't maintain that.

Can you run warpinator in debug mode and attach the log:
```
warpinator --debug
```
or if it's a flatpak:
```
flatpak run org.x.Warpinator --debug
```

**Comment by jtcraigjr** [comment]
yes, both the phone and the laptop see the other by the IP address, so the WiFi is up.
but for some reason, the phone is waiting for permission.  and shows "null@null" 192.168.1.6:42000

![Warpinator_Laptop_Workinig](https://user-images.githubusercontent.com/34912170/118552349-7257e280-b724-11eb-8b54-9e69083027f2.png)



![Phone Error jpg](https://user-images.githubusercontent.com/34912170/118552438-91ef0b00-b724-11eb-8e74-107665020eb5.jpg)

**Comment by jtcraigjr** [comment]
and the deBug code didn't seem to work?


`john@FX705GM:~$ warpinator --debug
Unknown option --debug
john@FX705GM:~$ flatpak run org.x.Warpinator --debug
error: app/org.x.Warpinator/x86_64/master not installed
john@FX705GM:~$ 
`

**Comment by mtwebster** [comment]
Oh yeah that's way too old.

You get an error running --debug with the flatpak also?
```
flatpak install --system org.x.Warpinator
flatpak run org.x.Warpinator --debug
```

**Comment by jtcraigjr** [comment]
I wished I had saved the entire output of Terminal.
Opening it up again, the only useful information is from the Version info:


~$ flatpak --version
Flatpak 1.11.1
john@FX705GM:~$ 


My friend Andy was able to get his working with a Samsung Android phone, but noted some issues with it.

https://forums.linuxmint.com/viewtopic.php?p=2016804#p2016804

I was advised by the Linux Mint forum users to post up my situation, and see it can be resolved easily??
maybe it is a problem with just my phone model?
It is on Android Pie 9 and last updated by Version last week.

.
